README.md

```diff
@@ -0,0 +1,21 @@
+# Generic connects
+
+This project contains extensions for base "[Connect](https://github.com/th2-net/th2-conn)" component
```

Comment:
This project contains extensions for **the base component** "[Connect](https://github.com/th2-net/th2-conn)"